packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -923,7 +904,7 @@ exports[`Podcast Promo Examples on-page example 1`] = `
       </div>
     </div>
     <div
-      class="emotion-40 emotion-41"
+      class="emotion-38 emotion-39"
     >
       <div
         class="emotion-4 emotion-1"
```

Comment:
what's the height for?

Reply:
This is the height of some of the dummy content used in the `on page` Storybook fixture. The height is just assigning how much height the element should take up...